src/pages/ContractInfoPage/components/ContractInfo/ContractInfoHeader.vue

```diff
@@ -47,6 +48,12 @@ const iconName = computed(() =>
     ? ICON_NAMES.ethereumAlt
     : ICON_NAMES.arbitrumAlt,
 )
+
+const projectClassName = computed(() =>
+  props.type === CONTRACT_INFO_NAMES.read
+    ? 'contract-info-header__project'
+    : 'contract-info-header__project contract-info-header__project--write',
+)
```

Comment:
For 1 conditional class it is ok, but if in future more conditional classes will be added it's better to use another approach.

```ts
const projectClassNames = computed(() =>
    [
        'contract-info-header__project',
        ...(props.type === CONTRACT_INFO_NAMES.read ? ['contract-info-header__project--write'] : [])
    ].join(' ')
)
```

Reply:
done